buildSrc/src/main/groovy/io/vantiq/extsrc/assy/tasks/AssemblyResourceGeneration.groovy

```diff
@@ -144,6 +162,11 @@ class AssemblyResourceGeneration extends DefaultTask {
         kameletJarFilename = kameletJarFilename.substring(0, kameletJarFilename.indexOf('!/'))
         kameletJarFilename = kameletJarFilename.substring(KAMELETS_RESOURCE_PATH.length())
 
```

Comment:
Capture version from jar artifact.  Transform it to be a valid package segment/vantiq entity name by replacing periods with underscores.